DuckDuckGo/Favicons.swift

```diff
@@ -67,34 +64,6 @@ public class Favicons {
         NotificationCenter.default.removeObserver(Constants.tabsCache)
     }
 
-    public func migrateFavicons(to size: CGSize, afterMigrationHandler: @escaping () -> Void) {
```

Comment:
It was agreed to remove this, right?

Reply:
yes

Reply:
see: https://app.asana.com/0/0/1209225275889527/f